test/libsolidity/util/TestFileParser.cpp

```diff
@@ -558,7 +558,7 @@ void TestFileParser::Scanner::scanNextToken()
 			else if (isWhiteSpace(current()))
 				token = selectToken(Token::Whitespace);
 			else if (isEndOfLine())
-				token = selectToken(Token::EOS);
+				token = make_pair(Token::EOS, "EOS");
```

Comment:
Why `"EOS"`?

Reply:
`EOS` = "End of String", which is IMHO a well known term for indicating the end of a string (which in this case is true). We could have used `EOL` for end-of-line, too. I just wanted to avoid writing "EOF" because that felt wrong. and when looking into the implementation of `selectToken`, I noticed that it is indeed adding a string-representation to `token.second` of the `token.first` value, when no literal was given, which is the case for "end of string/line" condition.

Reply:
Ah ok! This parser works a little different from the Solidity parser which I think would not have used a string in that case.